Source/Scene/Expression.js

```diff
@@ -62,7 +62,16 @@ define([
         asin : Math.asin,
         atan : Math.atan,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        degrees : CesiumMath.toDegrees,
+        sign : Math.sign,
+        floor : Math.floor,
+        ceil : Math.ceil,
+        round : Math.round,
```

Comment:
I think `round` can be treated like the other unary fns in that section. Only the shader area needs to have the special case.